ignite/engine/engine.py

```diff
@@ -776,7 +776,7 @@ def _internal_run(self) -> State:
             # update time wrt handlers
             self.state.times[Events.COMPLETED.name] = time_taken
             hours, mins, secs = _to_hours_mins_secs(time_taken)
-            self.logger.info(f"Engine run complete. Time taken: {hours:02d}:{mins:02d}:{secs:02d}")
+            self.logger.info(f"Engine run complete. Time taken: {hours:02d}:{mins:02d}:{secs:06.3f}")
```

Comment:
I do not get `6` in `06.3f`. seconds does not outnumber 60 which has two digits. why have you assumed 6 digits for it?

Reply:
Yes, `06.3f` means 6 digits in total for float number of seconds. Check for example:
```python
x = 7.890
print(f"{x:06.3f}")
```
>
```
07.890
```

I need the first padding zero to have uniform representation + milliseconds